src/components/numeral-date/numeral-date.pw.tsx

```diff
@@ -0,0 +1,711 @@
+import React from "react";
+import { test, expect } from "@playwright/experimental-ct-react17";
+
+import NumeralDate, { NumeralDateProps } from ".";
+import {
+  NumeralDateComponent,
+  NumeralDateControlled,
+} from "./components.test-pw";
+
+import {
+  numeralDateComponent,
+  numeralDateInput,
+} from "../../../playwright/components/numeral-date";
+
+import Box from "../box";
+
+import {
+  fieldHelpPreview,
+  getDataElementByValue,
+  getDataComponentByValue,
+  getDataRoleByValue,
+  tooltipPreview,
+} from "../../../playwright/components";
+
+import {
+  checkAccessibility,
+  verifyRequiredAsteriskForLabel,
+  assertCssValueIsApproximately,
+} from "../../../playwright/support/helper";
+
+import { ICON } from "../../../playwright/components/locators";
+import {
+  CHARACTERS,
+  SIZE,
+  VALIDATION,
+} from "../../../playwright/support/constants";
+
+const testData = [CHARACTERS.DIACRITICS, CHARACTERS.SPECIALCHARACTERS];
+
```

Comment:
Should these test be within a `test.describe`? The cypress tests were broken up into prop, event and accessibility tests.